### PR TITLE
Bootstrap electron

### PR DESCRIPTION
**Pull request by dewyze**
This commit adds a gitignore, nvmrc, ignores node modules, and a basic
start up `main.js` for electron.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "polyglot",
+  "version": "0.0.1",
```
Comment (crookedneighbor) [inline package.json]: You're going to want to add `"private": true,`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "repository": {
+      "type": "git",
+      "url": "git+https://github.com/dewyze/polyglot.git"
+  },
+  "license": "MIT",
+  "bugs": {
+      "url": "https://github.com/dewyze/polyglot/issues"
+  },
+  "devDependencies": {
+      "electron": "1.3.6"
```
Comment (crookedneighbor) [inline package.json]: 1.4.0 is the latest, fyi

Comment (dewyze) [inline package.json]: It is? Their releases page just says 1.3.6: http://electron.atom.io/releases/

File: package.json (hunk 1 above)
Comment (dewyze) [inline package.json]: What does that do?

Comment (crookedneighbor) [inline package.json]: It prevents you from accidentally publishing to the npm registry.

File: package.json (hunk 2 above)
Comment (crookedneighbor) [inline package.json]: https://www.npmjs.com/package/electron Shows 1.4.0 so they probably didn't push the tags on Github.

**Pull request merged**